### ISSUE TITLE
Aren't mattocks kinda OP?

### ISSUE THREAD

**Issue by ekiplier** [alcatrazEscapee/no-tree-punching#191]
I think the all-in-one idea is cool, but if there are no drawbacks, what's the point of crafting an axe/hoe/pickaxe? I think the mattock defeats the purpose of other tools, which is why I'm suggesting nerfing the tool (making it less efficient or less durable than other tools)

**Comment by Derpercoaster** [comment]
The very idea of a mattock is to replace the other tools by simply being able to do their jobs (they do exist in real life after all, for the same purpose). ANd if you think about it, the durability IS lower, in a way. The durability between it and a pickaxe is the same, but if you use it as a pickaxe, hoe, AND axe, the durability that could have been spent on mining stone was also spent on tilling soil and chopping wood, which means it was (in a way, at least) wasted.

In short, it has apparent flaws, but if you actually look into those supposed flaws there are really no issues. You are given a choice to decide where the durability it spent.

**Comment by alcatrazEscapee** [comment]
The durability per ingot spent is lower, yes.

Additionally, I added them specifically because in a mod where you *have* to use the right tool otherwise you literally can't mine blocks, it's much more important to have every type of tool on you. (As in vanilla, you can go without a shovel or hoe for example, and be fine.)

So no, I don't think they're particularly OP. At least, not in any way that any stats nerf would make any sense. If you don't want them, you can quite simply disable their crafting recipe and be free of them.
